configs/nginx/timenow.local

```diff
@@ -0,0 +1,23 @@
+server {
+	listen 443; 
+	listen [::]:443;
+
+	root /var/www/timenow; 
```

Comment:
According to the README.md documentation of this app - static files on the VM will be stored in `/var/www/timenow-fe` location. So this path has to be changed here.